.drone.yml

```diff
@@ -8,23 +8,23 @@ platform:
 
 steps:
 - name: build
-  image: cryptocom/chain-test:v1.1.1
+  image: cryptocom/chain-test:latest
```

Comment:
We need to increment the version number after pushed the image.

Reply:
It does not matter, Tomas can cover for that.